sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -589,6 +619,11 @@ public void run() {
         }, updateCurrentActivityRunnable, updateCurrentActivityRunnable);
     }
 
+    @Override
+    protected int getTriggerInterval() {
```

Comment:
Change trigger interval (using refactor options) to return a long and remove the cast.
The channel group already accepts a long.